tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_and_static_answers.spec.js

```diff
@@ -63,32 +62,31 @@ describe("Calculated summary with repeating answers", function () {
 
   it("Given I click on a change link, when I use the previous button, I return to the calculated summary", async () => {
     await dynamicAnswerChangeLink(1).click();
-    await expect(await browser.getUrl()).to.contain(DynamicAnswerPage.pageName);
```

Comment:
Think this was deleted as part of fix2 but missed when reverting it?

Reply:
Yeah, reverted these now to original state (except clicks conversion) plus some extra clicks were added in @berroar PR that merged and I had to change them too, applies to all the comments below.